CHANGELOG.rst

```diff
@@ -1,5 +1,26 @@
 .. _changelog:
 
+0.57.2
+------
+
+Renku ``0.57.2`` fixes several bugs in gateway and the `csi-rclone` driver.
+
```

Comment:
```suggestion
User-facing Changes
~~~~~~~~~~~~~~~~~~~

**Bug Fixes**

- **UI**: show the correct repository access status 
- **Sessions**: allow paused sessions with cloud storage secrets to resume normally

```

Reply:
both of the bug fixes were for issues users saw, so I think we should mention it at the top